woocommerce/wc-smooth-generator
# Shipping billing addresses feature

### All Submissions:






### Changes proposed in this Pull Request:



Closes #70. 

I have added more functionality than described in the issue #70. This needs a bit further explanation. I have divorced any shipping / billing address in three parts:
- person (firstname, lastname)
- address (address1,address2, city, state, postcode, country)
- company (company_name)

Every customer, who gets generated, has those three parts. Every part has a random generation. I added the feature that at the customer generation in Customer.php:
possibility A: billing = shipping (was done before)
possibility B: 1/2 billing != shipping (added with this pull request)

therefore, the shipping address can have different details for
- personal information (other person / same person)
- location information (other location / same location)
- company name (B2B (same company), B2B (different companies), B2C, C2B, C2C)

every part has its own random generation. with all three parts coming together its a unique order with in total (2*2*5) 40 combinations and therefore test cases.

### How to test the changes in this Pull Request:

1.use `wp wc generate customers 50` to directly test the customer generation 
2.use `wp wc generate orders 50` to see if the customers were correctly saved to the orders

### Other information:




### Changelog entry

> Enter a summary of all changes on this Pull Request. This will appear in the changelog if accepted.

### FOR PR REVIEWER ONLY:
